.drone.yml

```diff
@@ -0,0 +1,19 @@
+kind: pipeline
+name: default
+
+steps:
+- name: trigger  
+  image: plugins/downstream
+  fork: true
+  settings:
+    server: https://zkopru.sifnoc.net
+    token:
+      from_secret: drone_token
+    repositories:
+      - ${DRONE_REPO_OWNER}/stress-test
```

Comment:
should I set up the env variable?

Reply:
No, this variable is automatically inserted by a drone server.

Reply:
https://docs.drone.io/pipeline/environment/reference/drone-repo-owner/

Reply:
this one makes that the trigger is not affected by a forked repo.
and forked repo can use this drone config without edit.